contracts/bridge/contracts/BasicInboundChannel.sol

```diff
@@ -94,11 +87,12 @@ contract BasicInboundChannel {
     function processMessages(Message[] memory messages) internal {
         for (uint256 i = 0; i < messages.length; i++) {
             // Check message nonce is correct and increment nonce for replay protection
-            require(messages[i].nonce == nonce + 1, "invalid nonce");
+            require(messages[i].nonce == nonce + 1, "Channel: invalid nonce");
 
             nonce = nonce + 1;
 
             // Deliver the message to the target
+            // TODO: how to try catch this
             (bool success, ) =
                 messages[i].target.call{value: 0, gas: MAX_GAS_PER_MESSAGE}(
                     messages[i].payload
```

Comment:
maybe it is better to define an interface to call the target.

Reply:
A common `interface` may be a good idea to catch the external call  error.